.travis.yml

```diff
@@ -91,8 +91,7 @@ matrix:
             - txt2tags
             - xvfb
             - flip
-            - gcc-6
-            - g++-6
+            - clang-3.8
```

Comment:
Just curious: what is the advantage in using clang vs gcc?
Is it expected to be faster?
Why do you want to use it only for Qt5?